### PR TITLE
Added example for MelScale transform

### PR DESCRIPTION
**Pull request by ravimakhija**
Added example for MelScale transform as mentioned in issue #1564.

### CODE REVIEW

File: torchaudio/transforms/_transforms.py [hunk 1]
```diff
@@ -355,6 +355,13 @@ class MelScale(torch.nn.Module):
             (area normalization). (Default: ``None``)
         mel_scale (str, optional): Scale to use: ``htk`` or ``slaney``. (Default: ``htk``)
 
+    Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
```
Comment (nateanl) [inline torchaudio/transforms/_transforms.py:359]: The previous codes use single quotation marks for `test.wav`, they should be changed to double quotation marks to fit linter check. Let's also use double quotes in this PR.

File: torchaudio/transforms/_transforms.py [hunk 2]
```diff
@@ -355,6 +355,13 @@ class MelScale(torch.nn.Module):
...
+        >>> spectrogram_transform = torchaudio.transforms.Spectrogram(n_fft=1024)
```
Comment (nateanl) [inline torchaudio/transforms/_transforms.py:360]: for code consistency,
```suggestion
        >>> spectrogram_transform = transforms.Spectrogram(n_fft=1024)
```

**Review by nateanl — APPROVED** [review]
Thanks for working on it! The PR LGTM, just some nits to address.

File: torchaudio/transforms/_transforms.py (hunk 1 above)
Comment (ravimakhija) [reply torchaudio/transforms/_transforms.py:359]: Sounds good! I'm happy to do a sweep through other examples to change to double quotes. If there are any other code consistency thoughts you have, please let me know and I will incorporate them as well.

Comment (ravimakhija) [reply torchaudio/transforms/_transforms.py]: Changed to double quotes. ...will plan to use a separate PR to change double quotes for other transform examples.

Comment (nateanl) [reply torchaudio/transforms/_transforms.py]: I made a draft PR for changing double quotes https://github.com/pytorch/audio/pull/2618. That should fix the ones in functional/transform. Feel free to open a PR for other modules.

Comment (ravimakhija) [reply torchaudio/transforms/_transforms.py]: Ahh got it, thanks for the heads up!

**Pull request closed without merging**